keps/sig-api-machinery/00xx-admission-webhooks-to-ga.md

```diff
@@ -516,6 +627,16 @@ type Webhook struct {
      // on admission requests for ValidatingWebhookConfiguration and MutatingWebhookConfiguration objects.
      Rules []RuleWithOperations `json:"rules,omitempty" protobuf:"bytes,3,rep,name=rules"`
 
+     // ConversionPolicy describes whether the webhook wants to be called when a request is made 
+     // to a resource that does not match "rules", but which modifies the same data 
+     // as a resource that does match "rules".
+     // Allowed values are Ignore or Convert. 
+     // "Ignore" means the request should be ignored.
+     // "Convert" means the request should be converted to a kind corresponding to a resource matched by "rules", and sent to the webhook.
+     // Defaults to Convert.
```

Comment:
Is there a solid use case for ever setting it to ignore? It seems super backwards compatible, do we even need this option? (I didn't reread the whole kep, maybe the reasoning exists and I missed it.)

Reply:
we use per-version defaulting and admission logic in rest handlers today, this gives equivalent power to out-of-tree admission